src/app/store/collectibles/state.ts

```diff
@@ -0,0 +1,62 @@
+import type { UserObjktCollectible } from 'lib/apis/objkt';
+import { createEntity, LoadableEntityState } from 'lib/store';
+
+export interface CollectibleDetails extends Pick<UserObjktCollectible, 'fa' | 'description' | 'mime'> {
+  metadataHash: string | null;
+  /** Minted on date.
```

Comment:
It seems to me that this comment should be moved closer to `mintedTimestamp` property definition